src/classes/GE_DefaultTemplate.cls

```diff
@@ -0,0 +1,234 @@
+                                                    '"isRequiredFieldDisabled":true,"label":"Contact Matching Rule","required":true},{"allowDefaultValue":true,"apiName":' + 
+                                                    '"GiftBatch__c","dataType":"Boolean","isRequiredFieldDisabled":true,"label":"Gift Batch","required":true},{"allowDefaultValue"' + 
+                                                    ':true,"apiName":"Name","dataType":"String","isRequiredFieldDisabled":true,"label":"Batch Name","required":true},{"allowDefaultValue"' + 
+                                                    ':true,"apiName":"Process_Using_Scheduled_Job__c","dataType":"Boolean","isRequiredFieldDisabled":true,"label":"Process Using ' + 
+                                                    'Scheduled Job","required":true},{"allowDefaultValue":true,"apiName":"RequireTotalMatch__c","dataType":"Boolean",' + 
+                                                    '"isRequiredFieldDisabled":true,"label":"Require Expected Totals Match","required":true},{"allowDefaultValue":true,' + 
+                                                    '"apiName":"Contact_Matching_Rule__c","dataType":"String","isRequiredFieldDisabled":true,"label":"Contact Matching Rule"' + 
+                                                    ',"required":true},{"allowDefaultValue":true,"apiName":"GiftBatch__c","dataType":"Boolean","isRequiredFieldDisabled":true,' + 
+                                                    '"label":"Gift Batch","required":true},{"allowDefaultValue":true,"apiName":"Name","dataType":"String","isRequiredFieldDisabled"' + 
+                                                    ':true,"label":"Batch Name","required":true},{"allowDefaultValue":true,"apiName":"Process_Using_Scheduled_Job__c","dataType"' + 
+                                                    ':"Boolean","isRequiredFieldDisabled":true,"label":"Process Using Scheduled Job","required":true},{"allowDefaultValue":true,"apiName"' + 
+                                                    ':"RequireTotalMatch__c","dataType":"Boolean","isRequiredFieldDisabled":true,"label":"Require Expected Totals Match","required"' + 
+                                                    ':true}],"layout":{"fieldMappingSetDevName":"Migrated_Custom_Field_Mapping_Set","sections":[{"defaultDisplayMode":"expanded"' + 
+                                                    ',"displayRule":"displayRule","displayType":"accordion","elements":[{"customLabel":"Donor Type","dataImportFieldMappingDevNames"' + 
+                                                    ':["Donation_Donor__c"],"dataType":"PICKLIST","elementType":"field","id":"8ffa6e93-fd7b-2761-2c38-c466f1db98ec","label":' + 
+                                                    '"Data Import: Donation Donor","required":true,"sectionId":"6bc77bf4-9949-5b8b-9f2c-efe5663ef052"},{"customLabel":' + 
```

Comment:
@KietTruongSFDO Is there a label override missing here? We'd like this field to read Donor Type on form render.